feat:Add li to ul

nakagawamai opened the pull request:
# [Issue No.01](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#1%E3%81%93%E3%81%AEdom%E3%82%92html%E5%86%85%E3%81%AEul%E3%81%AE%E4%B8%AD%E3%81%AB%E5%B7%AE%E3%81%97%E8%BE%BC%E3%82%93%E3%81%A7%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84)
このDOMをhtml内のulの中に差し込んでください
`<li>これです</li>`

## [CodeSandBox](https://codesandbox.io/s/stupefied-joana-of8lp?file=/script.js)
[Latest Commit](https://github.com/nakagawamai/js-lesson/pull/2)

## Uneasy part
- I was worried that it might have been better to add id to ul.

nakagawamai committed:
feat:js-lesson01

create index.html,script.js

kenmori commented on script.js:
```diff
@@ -0,0 +1,2 @@
+var targetElements = document.getElementsByTagName('ul');
+targetElements[0].insertAdjacentHTML('afterbegin','<li>これです</li>');
```
IMO
If it is a tag name, there is another ui, and it will break when the order changes, so let's do id?

nakagawamai replied:
I appreciate the advice.I try it id.

kenmori commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Can this file be ignored with Git ignore?

nakagawamai replied:
Yes. I will create .gitignore and this file ignore.

nakagawamai committed:
style:Add id to ul.

to get the element from id.

nakagawamai commented on .gitignore:
```diff
@@ -0,0 +1 @@
+.vscode/
```
Create .gitignore to ignore .vscode/settings.json.

nakagawamai commented on index.html:
```diff
@@ -9,7 +9,7 @@
   <body>
     <div>
-      <ul></ul>
+      <ul id="list"></ul>
```
Add id to ul to get the element from id.

nakagawamai committed:
update:Change variable name.

nakagawamai commented on script.js:
```diff
@@ -1,2 +1,2 @@
-targetElements.insertAdjacentHTML('afterbegin','<li>これです</li>');
\ No newline at end of file
+var targetElement = document.getElementById('list');
+targetElement.insertAdjacentHTML('afterbegin','<li>これです</li>');
```
Changed the variable name, because it was plural.

nakagawamai committed:
remove:remove Delete unnecessary code.

nakagawamai commented on index.html:
```diff
@@ -4,7 +4,6 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" type="text/css" />
```
this is unnecessary.

443n0511 commented on index.html:
```diff
@@ -0,0 +1,15 @@
+    </div>
+  </body>
+  <script src="script.js"></script>
+</html>
```
MUST
Please make a new line at the end of the sentence.
You might want to review your editor settings.

[コードの行末を改行してないとGithubで注意されるのは、editorの設定で防ぐ✏️【Android Studio】](https://qiita.com/syuheifujita/items/960754d0c821f9f6407c)

nakagawamai replied:
Thank you for informing me very mush!
I set the editor.

nakagawamai committed:
change:Changed variable declaration from var to const.

var is easy to rewrite variable.
Changed from var to const as it can cause unintended bugs.

nakagawamai committed:
update:Insert a blank line at the end of the sentence.

nakagawamai commented on script.js:
```diff
@@ -0,0 +1,2 @@
+var targetElement = document.getElementById('list');
```
Thank you review so much!
Because I knew only var. I looked up the variable declaration and modified it to const.
`const targetElement = document.getElementById('list');`
[Latest](https://github.com/nakagawamai/js-lesson/pull/2/commits/fe68ca1ab9be2c654c1d3786f548e9cfc6da851a)

443n0511 replied:
It will be very useful for future assignments to look into the difference between let const ver! Please look into it!

nakagawamai replied:
I find out about the difference in var let const.
Thank you so much!

nakagawamai commented on script.js:
```diff
@@ -1,2 +1,2 @@
-var targetElement = document.getElementById('list');
+const targetElement = document.getElementById('list');
```
var is easy to rewrite variable.
Changed from var to const as it can cause unintended bugs.

nakagawamai commented on index.html:
```diff
@@ -12,4 +12,4 @@
   <script src="script.js"></script>
-</html>
\ No newline at end of file
+</html>
```
Insert a blank line at the end of the sentence.

443n0511 commented on script.js:
```diff
@@ -0,0 +1,2 @@
+const targetElement = document.getElementById('list');
+targetElement.insertAdjacentHTML('afterbegin','<li>これです</li>');
```
MUST
Let's insert a blank line here as well.

nakagawamai replied:
Thank you for informing me very mush!
I set the editor.

Insert a blank line at the end of the sentence.

nakagawamai committed:
Insert a blank line at the end of the sentence.

nakagawamai commented on script.js:
```diff
@@ -1,2 +1,2 @@
 const targetElement = document.getElementById('list');
-targetElement.insertAdjacentHTML('afterbegin','<li>これです</li>');
\ No newline at end of file
+targetElement.insertAdjacentHTML('afterbegin','<li>これです</li>');
```
Insert a blank line at the end of the sentence.

yoshikawa-akr reviewed (approved):
I've never heard of the `insertAdjacentHTML` method before.
Thanks for letting me learn through your review!
I think the rest of the code is fine.

nakagawamai commented:
@kenmori @443n0511 @yoshikawa-akr
Thank you for your approval!
